src/plots/gl3d/layout/tick_marks.js

```diff
@@ -47,7 +47,9 @@ function computeTickMarks(scene) {
         axes._length = (glRange[i].hi - glRange[i].lo) *
             glRange[i].pixelsPerDataUnit / scene.dataScale[i];
 
-        if(Math.abs(axes._length) === Infinity) {
+        if(Math.abs(axes._length) === Infinity ||
+           Math.abs(axes._length) === -Infinity ||
```

Comment:
wait, how can `Math.abs(...) === -Infinity`?

Reply:
Thanks for the note. Just removed that line.

Reply:
Thanks. Anyway much better to fix the issue here than in `axes.js`!

💃